src/systemcmds/pwm/pwm.c

```diff
@@ -122,7 +122,7 @@ get_parameter_value(const char *option, const char *paramDescription)
 	if (strncmp("p:", option, 2) == 0) {
 
 		char paramName[32];
-		strncpy(paramName, option + 2, 16);
+		strncpy(paramName, option + 2, 17);
```

Comment:
Is 17 the maximum length of a parameter name? 
Everything else looks good to me.